src/main/java/ru/curs/lyra/service/LyraService.java

```diff
@@ -112,42 +130,40 @@ public JSONObject getMetadata(CallContext callContext, FormInstantiationParamete
         for (LyraFormField field : lyraFields.values()) {
 
             JSONObject column = new JSONObject();
-            column.put("id", field.getName());
-            column.put("caption", field.getCaption());
+            column.put(ID, field.getName());
+            column.put(CAPTION, field.getCaption());
 
-            column.put("visible", field.isVisible());
+            column.put(VISIBLE, field.isVisible());
 
-            column.put("cssClassName", field.getCssClassName());
-            column.put("cssStyle", field.getCssStyle());
+            column.put(CSS_CLASS_NAME, field.getCssClassName());
+            column.put(CSS_STYLE, field.getCssStyle());
 
             if (lyraGridAvailableSorting.contains(field.getName())) {
-                column.put("sortingAvailable", "true");
+                column.put(SORTING_AVAILABLE, true);
             }
 
             columns.put(String.valueOf(++count), column);
 
         }
 
-        metadata.put("columns", columns);
+        metadata.put(COLUMNS, columns);
 
         return metadata;
 
     }
 
 
     private Object[] getKeyValuesById(final String refreshId) {
-        return refreshId.split(KEYVALUES_SEPARATOR);
+        JSONArray jsonArray = new JSONArray(refreshId);
+        List<Object> list = new ArrayList<>();
```

Comment:
Why using List and then converting it to Array?
We know the `jsonArray.length` in advance, so we could just create `new Object[jsonArray.length()]`, populate it in cycle and return it.